packages/core/src/model.ts

```diff
@@ -118,6 +44,7 @@ export function createModel(initialContext: object, creators?): unknown {
     reset: () => assign(initialContext),
     createMachine: (config, implementations) => {
       return createMachine(
+        // @ts-ignore
```

Comment:
I've tried removing this locally and it seems that it works just OK without this ignore comment

Reply:
d3413bb2f75562ea39f1d2301b8261821368e9ae